### DISCUSSION TITLE
Newbie question

### DISCUSSION THREAD

**Discussion by 40federico** [unifi-utilities/unifi-common#296] [Q&A]
i am happy to see through this work is possible to add new features to the UDMPro. But not sure if one of my critical question is solved. I mean I need 3 VLANs on the Wan side due how works my ISP (internet, Voip and Multicast IPTV). I had an ER-PoE router where it was possible. I have the code and so on to have it up and running. But is it possible here?

**Comment by AmyYummy** [comment]
It is possible, but it is recommended to ask your POE [producer](www.baudcom.cn)
